theme/templates/includes/author-link.html

```diff
@@ -0,0 +1,18 @@
+<a data-toggle="modal" class="author-preview"
```

Comment:
Isn't this an example of an html snippet using django templates?  (An alternative to using dominate or pure JS)

Reply:
So you're in favor or django snippets instead of the alternate solutions we have scattered through the system?

Reply:
Indeed. This offers great re-usability through Django template language, while providing all the flexibility of writing HTML directly.

Reply:
In my opinion you can do more with Dominate than what you can do with django template. For example, instead of template filters you can use python to do all those magic. You have full power of python for generating html code. The html generated using dominate tags goes through django template engine before it is sent to UI. You can do whatever re-usability you can get in template language using dominate.  The only disadvantage I see using dominate is that front-end developers not familiar with python may find it difficult to maintain. For that reason only, I would agree that we should replace dominate code with django template code.